docs/source/advanced/no-runtime.md

```diff
@@ -26,9 +28,31 @@ If you do have custom GraphQL scalar types, pass properly configured instance of
 
     ScalarTypeAdapters scalarTypeAdapters = new ScalarTypeAdapters(<provide your custom scalar type adapters>);
 
-    Response<Query.Data> response = new Query().parse(httpResponse.body().source(), scalarTypeAdapters);
+    Response<Operation.Data> response = new Query().parse(httpResponse.body().source(), scalarTypeAdapters);
+```
+
+## Converting Query.Data back to JSON
```

Comment:
Does that belong to the `no-runtime` doc ? I think the original use case was for testing with a mock server. Not sure where to put it though... 😅

Reply:
Suggestions are welcome. All of the stuff mentioned in this doc are in api module rather than runtime. I didn't really want to create a new file.

Reply:
Should I move out?

Reply:
I don't really have a better idea where to put this doc. Maybe at some point do a 'testing' page ? But I guess `no-runtime` is fine for now, sorry for the noise.

Reply:
Not at all. It's useful feedback.